### ISSUE TITLE
[CAL-404] Dynamic Link issue list

### ISSUE THREAD

**Issue by alishaz-polymath** [calcom/cal.com#5597]
Currently, Dynamic Links are functional but not polished. Here's a list of issues we need to sort out before we can consider it a finished feature.

**Critical**

* [X] The availability should be fetched as a collective (which is not the case currently). This is by far the **most critical** issue as it often suggests either invalid times for teams in different time zones, or blocks out valid times despite them being available for all users in the list

**Important**

* [X] **Improving subject line:** The Name isn't listed in the subject like as we make use of username. We should fetch the names of users from the username and use them in the subject line of booking emails.

  [It still shows up as Nameless](https://share.cleanshot.com/TYh271).
* [X] **Enable/disable dynamic booking links:** I couldn't find the toggle to disable dynamic booking links for any user (it used to reside in the user's settings and is a crucial setting for higher profile users who don't wish to engage in dynamic booking at all)

Please feel free to add more to these if I have missed any.

> From [SyncLinear.com](https://synclinear.com) | [CAL-404](https://linear.app/calcom/issue/CAL-404/dynamic-link-issue-list)

* \#5517
* \#3884

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-404](https://linear.app/calcom/issue/CAL-404/dynamic-link-issue-list)</sub>

**Comment by alishaz-polymath** [comment]
Also adding #5517 to this list

**Comment by alishaz-polymath** [comment]
Added #3884 to the list

**Comment by shirazdole** [comment]
Ali this is great. You covered everything that comes to mind. Was not aware of the **critical** portion… 

Eventually, it would be a great way to do "dynamic search" to find the usernames of those on your team, maybe cal.com/dynamic and then there's a text box that pulls from a database of usernames from your team based on your account so that it's easier to create the dynamic links

**Comment by Jaibles** [comment]
Yes we should probably split that out into a sept issue when tackling 👍ok for now.

**Comment by alishaz-polymath** [comment]
That's the plan @Jaibles As I keep creating a new ticket for each of the issues in the list, I'll start adding the link here to have a single source of all Dynamic Link related issues 🙏

**Comment by shirazdole** [comment]
- Cannot add guests on Dynamic Links
